buildspec.yml

```diff
@@ -0,0 +1,75 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_VERSION: "1.3.0"
+    ACCOUNT: "142577830533"
```

Comment:
this means we can't test this build in another account. we shouldn't need to specify this account directly.

Reply:
Updated. Now we extract account number from codebuild env variable for arn.

Reply:
might be more straightforward to do this:

`AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query 'Account' --output text)`

Reply:
Updated! Thx